envi/archs/arm/disasm.py

```diff
@@ -39,7 +39,16 @@
 #   * VectorPointFloat subsystem (coproc 10+11)
 #   * Debug subsystem (coproc 14)
 #   * other 'default' coprocs we can handle and add value?
-    
+
```

Comment:
nope.  please make this a field in the Disasm class.  make it stored as a non-string (ie.  store the bitmask).  they accessors are good (get/set) but should also be a part of the Disasm class.  this allows us to set this for each instance of the disassembler, not globally.